test/mock/tribunals/data/paper/appealReceived.json

```diff
@@ -1,33 +1,31 @@
 {
     "appeal": {
-      "caseReference": "SC285/18/00002",
-      "name": "Mrs. AN Test",
-      "surname": "Test",
-      "appealNumber": "1CA27vtZ1p",
-      "benefitType": "esa",
-      "hearingType" : "paper",
-      "status": "APPEAL_RECEIVED",
-      "latestEvents": [
-        {
-          "date": "2017-03-08T00:00:00Z",
-          "type": "APPEAL_RECEIVED",
-          "dwpResponseDate": "2017-04-12T00:00:00Z",
-          "contentKey": "status.appealReceived"
+        "caseReference": "SC285/18/00002",
+        "name": "Mrs. AN Test",
+        "surname": "Test",
+        "appealNumber": "1CA27vtZ1p",
+        "benefitType": "esa",
+        "hearingType": "paper",
+        "createdInGapsFrom": "Ready to List",
```

Comment:
@chris-moreton , don't we need to update the mock data too? Where you able to run it locally and confirm it actually works?